geomesa-accumulo/geomesa-accumulo-dist/pom.xml

```diff
@@ -23,116 +29,87 @@
-            <artifactId>geomesa-accumulo-spark-runtime-accumulo1_${scala.binary.version}</artifactId>
-            <version>${project.version}</version>
+            <artifactId>geomesa-accumulo-gs-plugin_${scala.binary.version}</artifactId>
+            <type>pom</type>
+            <scope>provided</scope>
+            <exclusions>
+                <exclusion>
+                    <groupId>*</groupId>
+                    <artifactId>*</artifactId>
+                </exclusion>
+            </exclusions>
         </dependency>
         <dependency>
             <groupId>org.locationtech.geomesa</groupId>
-            <artifactId>geomesa-spark-converter_${scala.binary.version}</artifactId>
-        </dependency>
-
-        <!-- Yarn (dist)-->
-        <dependency>
-            <groupId>org.locationtech.geomesa</groupId>
-            <artifactId>geomesa-accumulo-jobs_${scala.binary.version}</artifactId>
+            <artifactId>geomesa-geojson-gs-plugin_${scala.binary.version}</artifactId>
+            <type>pom</type>
+            <scope>provided</scope>
+            <exclusions>
+                <exclusion>
+                    <groupId>*</groupId>
+                    <artifactId>*</artifactId>
+                </exclusion>
+            </exclusions>
         </dependency>
-
-        <!-- GeoServer Plugins (dist) -->
         <dependency>
             <groupId>org.locationtech.geomesa</groupId>
-            <artifactId>geomesa-accumulo-gs-plugin_${scala.binary.version}</artifactId>
+            <artifactId>geomesa-stream-gs-plugin_${scala.binary.version}</artifactId>
             <type>pom</type>
+            <scope>provided</scope>
+            <exclusions>
+                <exclusion>
+                    <groupId>*</groupId>
+                    <artifactId>*</artifactId>
+                </exclusion>
+            </exclusions>
         </dependency>
+
+        <!-- Spark (dist)-->
         <dependency>
             <groupId>org.locationtech.geomesa</groupId>
-            <artifactId>geomesa-process-wps_${scala.binary.version}</artifactId>
+            <artifactId>geomesa-accumulo-spark-runtime-accumulo2_${scala.binary.version}</artifactId>
+            <version>${project.version}</version>
         </dependency>
         <dependency>
             <groupId>org.locationtech.geomesa</groupId>
-            <artifactId>geomesa-stream-gs-plugin_${scala.binary.version}</artifactId>
-            <type>pom</type>
+            <artifactId>geomesa-accumulo-spark-runtime-accumulo1_${scala.binary.version}</artifactId>
+            <version>${project.version}</version>
         </dependency>
         <dependency>
             <groupId>org.locationtech.geomesa</groupId>
-            <artifactId>geomesa-geojson-gs-plugin_${scala.binary.version}</artifactId>
-            <type>pom</type>
+            <artifactId>geomesa-spark-converter_${scala.binary.version}</artifactId>
         </dependency>
 
-        <!-- Web Services (dist) -->
```

Comment:
Hmm... we were packing the web services in the Accumulo dist?  Was it the only bundle like that?

Reply:
yeah, it was the only one. and it doesn't seem relevant to include, since you'd generally pull it down through maven.